pkgs/os-specific/linux/decklink/default.nix

```diff
@@ -17,9 +18,24 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs =  kernel.moduleBuildDependencies;
 
+  patches = lib.optionals (lib.versionAtLeast kernel.version "6.8") [
+    (fetchpatch {
+      name = "decklink-addMutex.patch";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/01-addMutex.patch?h=decklink&id=132ce45a76e230cbfec4a3daac237ffe9b8a377a";
+      sha256 = "sha256-YLIjO3wMrMoEZwMX5Fs9W4uRu9Xo8klzsjfhxS2wRfQ=";
+    })
+    (fetchpatch {
+      name = "decklink-changeMaxOrder.patch";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/02-changeMaxOrder.patch?h=decklink&id=132ce45a76e230cbfec4a3daac237ffe9b8a377a";
+      sha256 = "sha256-/erUVYjpTuyaQaCSzSxwKgNocxijc1uNaUjnrJEMa6g=";
+    })
+  ];
+
+
   postUnpack = ''
-    tar xf Blackmagic_Desktop_Video_Linux_${lib.versions.majorMinor version}/other/${stdenv.hostPlatform.uname.processor}/desktopvideo-${version}-${stdenv.hostPlatform.uname.processor}.tar.gz
+    tar xf Blackmagic_Desktop_Video_Linux_${lib.head (lib.splitString "a" version)}/other/${stdenv.hostPlatform.uname.processor}/desktopvideo-${version}-${stdenv.hostPlatform.uname.processor}.tar.gz
```

Comment:
possibly extract the uname stuff to a let statement with separate variable for it, to improve clarity here.

Reply:
applied your suggestion.